docs/adr/20231212-subcutaneous-tests-should-target-webapi-layer.md

```diff
@@ -0,0 +1,42 @@
+# Subcutaneous Tests Should Target WebApi Layer
+
+- Status: accepted
+- Deciders: Daniel M, William L, Matt W
+- Date: 2023-12-12
+- Tags: tests
+
+Technical Story: https://github.com/SSWConsulting/SSW.CleanArchitecture/issues/201
+
+## Context and Problem Statement
+
+Previously, the Subcutaneous tests were targeting the Application Layer. This was ok, but skipped the WebApi layer, which is the layer that the end user will be interacting with. This means that the tests were not testing the full stack, and we were not getting the full benefit of the Subcutaneous tests.
+
+## Decision Drivers
+
+- Subcutaneous tests should test as much of the stack as possible
+- Some element of contract testing is required
+
+## Considered Options
+
+1. Leave the Subcutaneous tests targeting the Application Layer
+2. Change Subcutaneous tests to target the WebApi Layer
```

Comment:
I think there needs to be a mention that we decided to hard code the routes in the tests as opposed to using a tool generated wrapper.

We did this as a simple form of contract tests as well

Reply:
Done.